assets/license-header

```diff
@@ -0,0 +1,15 @@
+<https://github.com/radicle-dev/<project name>>
+
+<project name> is Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
+
+<project name> is licensed under version 3 of the GNU General Public License
+(GPLv3) with the Radicle Linking Exception. Any contribution to <project name> must be made subject to the developer certificate of origin (DCO).
```

Comment:
is there a reason as to why this line is not wrapped like all the others?

Reply:
I left it unformatted, but can wrap it too